packages/cli/src/databases/entities/execution-entity.ts

```diff
@@ -46,7 +46,20 @@ export class ExecutionEntity {
 	@Column('varchar')
 	status: ExecutionStatus;
 
-	@Column(datetimeColumnType)
+	/**
+	 * Time when the execution was created. This column is nullable to represent
+	 * missing data for executions created before this column was added.
+	 */
+	@Column({ type: datetimeColumnType, nullable: true })
```

Comment:
instead of making this nullable, should we instead copy over all the previous values from `startedAt`

Reply:
Interesting! `createdAt` and `startedAt` should always differ, so strictly speaking copying over wouldn't be correct. Is there more value in keeping this column non-nullable than in representing past executions accurately?

Reply:
Will do, thanks both

Reply:
[99188b4](https://github.com/n8n-io/n8n/pull/10810/commits/99188b498535c02cc80d8de4bcedbd094c2efcc0)